### PR TITLE
EXP-163 :Remover botão de reprocessamento de transação para clientes link.me na pilot

### PR DESCRIPTION
**Pull request by diegodsgarcia**
## Contexto
Esconde ação de "Reprocessamento" quando uma company é do tipo `payment_link`

## Checklist
- [x] Esconde botão "reprocessamento"

## Issues linkadas
- [x] [#EXP-163 :Remover botão de reprocessamento de transação para clientes link.me na pilot](https://mundipagg.atlassian.net/secure/RapidBoard.jspa?rapidView=185&modal=detail&selectedIssue=EXP-163)

### Preview
**COM PAYMENT LINK**
image

**SEM PAYMENT  LINK**
image

## Como testar?
- Entre em company do tipo `payment_link_app` e crie uma transação recusada (vc pode criar com o CVV 6) e veja se o botão de reprocessamento foi escondido

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: cf77400203e6afec8daaa5a0ae2107cb9e1565af
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: cf77400203e6afec8daaa5a0ae2107cb9e1565af
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: cf77400203e6afec8daaa5a0ae2107cb9e1565af
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: cf77400203e6afec8daaa5a0ae2107cb9e1565af
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/TransactionDetails/index.js [hunk 1]
```diff
@@ -344,7 +348,7 @@ class TransactionDetails extends Component {
           always(null)
         ),
         ifElse(
-          propEq('reprocessable', true),
+          both(propEq('reprocessable', true), isNotPaymentLink),
```
Comment (marisamodolo) [inline packages/pilot/src/containers/TransactionDetails/index.js:351]: O que acha de usarmos o `always`  no lugar do `complement` aqui? Assim sempre retornamos uma função com o valor de `!isPaymentLink` e não precisaria criar uma nova função. Ficaria algo como:
```js
both(propEq('reprocessable', true), always(!isPaymentLink))
```

Comment (vagnervst) [reply packages/pilot/src/containers/TransactionDetails/index.js:351]: Acho válida essa sugestão @diegodsgarcia, vai precisar fazer rebase depois do merge do PR do Ro, já pode subir com esse refactor se achar uma boa

**Pull request merged**